orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -245,6 +245,26 @@ def show_docs(self):
             vertical-align: top;
             padding-right: 10px;
         }}
+        
+        .content {{
+            /* Adopted from https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/ */
+        
+            /* These are technically the same, but use both */
+            overflow-wrap: break-word;
+            word-wrap: break-word;
+        
+            -ms-word-break: break-all;
+            /* This is the dangerous one in WebKit, as it breaks things wherever */
+            word-break: break-all;
+            /* Instead use this non-standard one: */
+            word-break: break-word;
+        
+            /* Adds a hyphen where the word breaks, if supported (No Blink) */
+            -ms-hyphens: auto;
+            -moz-hyphens: auto;
+            -webkit-hyphens: auto;
+            hyphens: auto;
+        }}
```

Comment:
This can be trimmed to three lines: overflow-wrap, word-wrap, word-break: break-all; hyphen.